packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/face-value/balance-view/index.hbs

```diff
@@ -0,0 +1,43 @@
+<details class="balance-view-banner" ...attributes>
+  <summary class="balance-view-banner__summary">
+    <CardPay::BalanceDisplay
+      class="balance-view-banner__summary-content"
+      @size="small"
+      @icon="avatar-default"
+      @name="Account 1"
+      @text={{concat "DEPOT: " @balance " " @token.symbol}}
+      data-test-balance-view-summary
+    />
+    <div class="balance-view-banner__marker">
+      {{svg-jar "caret-right" class="balance-view-banner__marker-icon" role="presentation"}}
+    </div>
+  </summary>
+  <CardPay::NestedItems class="balance-view-banner__details">
+    <:outer>
+      <CardPay::AccountDisplay
+        @size="small"
+        @icon="avatar-default"
+        @name="Account 1"
+        @address={{@walletAddress}}
+        data-test-balance-view-account-address
+      />
+    </:outer>
+    <:inner>
+      <CardPay::AccountDisplay
+        @size="small"
+        @icon="depot"
+        @name="DEPOT:"
+        @address={{@depotAddress}}
+        data-test-balance-view-depot-address
+      />
+      <CardPay::BalanceDisplay
+        class="balance-view-banner__details-token"
+        @size="large"
+        @icon={{@token.icon}}
+        @balance={{@balance}}
+        @symbol={{@token.symbol}}
+        data-test-balance-view-token-amount
+      />
+    </:inner>
+  </CardPay::NestedItems>
```

Comment:
NestedItems kind of breaks down at 3 levels. If 3 levels of nesting is necessary and comes up often (which it seems to?), may be good to update the component to accommodate this (as a separate issue).

Reply:
What breaks about it?

Reply:
Nothing in the styles directly.

I think having to nest NestedItems is probably not great, or adding extra styles to support 3 levels each time either. It would be nice to have the component support 3 levels directly I think? Though this is not a must and definitely not blocking for this PR